<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistFrame</class>
 <widget class="QWidget" name="RegistFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="edit_name">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_confirm">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_pw">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光仿宋_CNKI</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DBMS用户注册界面</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="dbaCheck">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>管理员</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="createCheck">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>创建权限</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="updateCheck">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>修改权限</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="deleteCheck">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>删除权限</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dbaCheck</sender>
   <signal>clicked()</signal>
   <receiver>RegistFrame</receiver>
   <slot>on_checkBox_dba()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_checkBox_dba()</slot>
 </slots>
</ui>
